### ISSUE TITLE
Testing with unitduino or UnixHostDuino will always exit on success

### ISSUE THREAD

**Issue by ricardomsmonteiro** [bxparks/AUnit#51]
No matter if tests fail, if the tests are ran in a CI/CD platform, like gitlab runner, it won't be able to detect a failure. I believe it's related with exit(0) instruction in TestRunner.cpp line 113.

**Comment by bxparks** [comment]
Good idea, done! Here's the testing that I performed:
```
$ cd tests/AUnitTest
$ make clean && make -j && ./AUnitTest.out
...
$ echo $?
0

$ cd tests/FailingTest
$ make clean && make -j && ./FailingTest.out
...
$ echo $?
1
```

**Comment by bxparks** [comment]
Fixed on 1.3.1
